spec/lib/hubspot/task_spec.rb

```diff
@@ -6,7 +6,13 @@
         hs_task_subject: 'title of task',
         hs_timestamp: DateTime.now.strftime('%Q')
       }
-      described_class.create!(params, ticket_id: 16_174_569_112)
+      associations = [
+        Hubspot::Association.build_association_param('Task', 'Ticket', 16_174_569_112),
+        Hubspot::Association.build_association_param('Task', 'Contact', 75_761_595_194),
+        Hubspot::Association.build_association_param('Task', 'Company', 25_571_271_600),
+        Hubspot::Association.build_association_param('Task', 'Deal', 28_806_796_888),
```

Comment:
Still wondering if we should do a helper method in Association that would receive something like
```ruby
Hubspot::Association.build_association_params('Task', ['Ticket', 16_174_569_112], ['Contact', 75_761_595_194], ...)
```
etc
that would do the multiple lines